sourcegraph/sourcegraph
# Wildcard V2: `<Menu />`, `<MenuItem />` Manual migration

## Description
Migrate the following to use `wildcard`'s `Menu*` components

## Ref 
[SG Issue](https://github.com/sourcegraph/sourcegraph/issues/32275)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-32275)

## Test plan
- Click to view attached deployment preview link
- Compare migrated components, to the production app
- Confirm there are no UI variations.

## Preview App
[Frontend](https://client-sourcegraph-frontend-pr-273.onrender.com/)
[Storybook](https://client-sourcegraph-storybook-pr-273.onrender.com/)


## Implementation Detail
From Reactstrap:
- `NotebookShareOptionsDropdown`

From Reach: Note we should also close this issue when done: https://github.com/sourcegraph/sourcegraph/issues/29435
- `InsightContextMenu`
- `DashboardMenu`
- `View.story.tx` and `ViewGrid.story.tsx`
- `ActivationDropdown` - We also should move this to `client/web/src/components` as it is only used by the web code.
